README.md

```diff
@@ -302,6 +302,36 @@ Passing in 0 disables exponential backoff completely. In that case, the client w
 
 
 ## Upgrading
+
+### v93 -> v94
+- An event's chart key can no longer be changed
+
+### v92 -> v93
+- Objects can be released in batch
+
+### v91 -> v92
+- Use new RELEASE statusChangeCommand type instead of hardcoded FREE status, when releasing objects
+- Add methods to put up objects for resale
+
+### v90 -> v91
```

Comment:
The idea was to list breaking changes here, so that customers know what to change when upgrading. No need to list all the smaller changes.